ui/widgets/top-bar/src/components/Topbar.tsx

```diff
@@ -81,8 +92,9 @@ const Topbar: React.FC<ITopbarProps> = props => {
           onClick={onBackClick}
         />
       </div>
-      <div
-        className={tw('p-0 m-0 cursor-pointer flex(& col) justify-center items-center')}
+      <Button
+        plain={true}
+        customStyle={tw('p-0 m-0 cursor-pointer flex(& col) justify-center items-center')}
```

Comment:
We need to include `cursor-pointer` inside `Button` component when `plain` is true :) I should have added this.  Can we use `Stack` component for aligning and justifying center ?

Reply:
I have updated it to use Stack. `cursor-pointer` is included as a custom-style

Reply:
Regarding, `cursor-pointer` is it really needed here because button's default behavior is to have a cursor pointer ? That's the part I am not clear with :)

Reply:
I checked and indeed the button's default behavior is to have a cursor pointer so the added style is not really needed. But I will remove it in another PR :)